### PR TITLE
Finished Task 2

### PR DESCRIPTION
**Pull request by zapataa0**
I am ready for my code to be reviewed so it can me integrated, thank you!

### CODE REVIEW

File: hw03_zapataa.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
+
```
Comment (sheggen) [inline hw03_zapataa.py:1]: Remove the blank line at the beginning.

File: hw03_zapataa.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+#
+#################################################################################
+# Acknowledgements:
+#https://docs.python.org/3/library/turtle.html
+#
+#################################################################################
+
+import turtle
+
+def drawHouse(t):
```
Comment (sheggen) [inline hw03_zapataa.py:18]: Pay attention to gray squigglies, they indicate style guidelines. This one says two blank lines before any function definitions.

File: hw03_zapataa.py [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+    t.left(180)
+    t.forward(10)
+    t.right(145)
+    t.forward(70)
+    t.right(75)
+    t.forward(65)
+    t.end_fill()
+
+    pass
+    # ....
```
Comment (sheggen) [inline hw03_zapataa.py:48]: Remove helper text from the template.

File: hw03_zapataa.py (hunk 1 above)
Comment (zapataa0) [reply hw03_zapataa.py]: @sheggen fixed

File: hw03_zapataa.py (hunk 2 above)
Comment (zapataa0) [reply hw03_zapataa.py:18]: fixed syle guidlines

File: hw03_zapataa.py (hunk 3 above)
Comment (zapataa0) [reply hw03_zapataa.py]: removed helper texted

**Pull request merged**